packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -112,10 +122,14 @@ const Cell = (props) => {
     }
   };
 
-  const handleMouseDown = () => {
+  const handleMouseDown = (event) => {
     setFocusedCell({ rowId, columnId: id });
     if (isSelectableCell && onCellSelect) {
-      onCellSelect({ rowId, columnId: id });
+      event.preventDefault();
+      if (onCellSelect) {
+        onCellSelect({ rowId, columnId: id });
+        cellRef?.current.focus();
```

Comment:
The component should get focus just because it was clicked.   I would not expect you to need this logic as long as the tab index is set properly.

Reply:
This line was added to fix the same issue I described above. When the featured icon was placed inside of a cell, even though the icon wasn't an interactive element, per click on the icon the cell would lose the focus. Adding `event.preventDefault()` and `cellRef?.current.focus()` resolved the problem.

Reply:
UPD: update explained here: https://github.com/cerner/terra-framework/pull/1950#discussion_r1433007127